Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 [dependencies]
 chrono = "0.4.38"
 dotenvy = "0.15"
-serenity = "0.12"
+serenity = "0.12.4"
```

Comment:
I purposefully don't specify the patch number, because I prefer for that to be left to the lock file. I should do the same for `chrono`

Reply:
I specified the patch number since in this case it would actually not build on `0.12.2` due to `.contexts` being stabilized in the discord api

Reply:
I did a cargo update and it pulled the latest version (0.12.4)